server/routes/bookAVisit.ts

```diff
@@ -123,9 +125,48 @@ export default function routes(
         return closedVisit || visitor.restrictions.some(restriction => restriction.restrictionType === 'CLOSED')
       }, false)
       visitSessionData.visitRestriction = closedVisitVisitors ? 'CLOSED' : 'OPEN'
-
       visitSessionData.closedVisitReason = closedVisitVisitors ? 'visitor' : undefined
 
+      const closedVisitPrisoner = visitSessionData.prisoner.restrictions.some(restriction => {
+        return restriction.restrictionType === 'CLOSED'
+      })
```

Comment:
Can actually do on 1 (2 after prettier) lines but I'm going to approve and merge anyways 🙂 

`const closedVisitPrisoner = visitSessionData.prisoner.restrictions.some(restriction => restriction.restrictionType === 'CLOSED')`

Reply:
I should have spotted that as I've got something similar a couple of lines above 🤦‍♂️!